src/js/common/parser.js

```diff
@@ -0,0 +1,223 @@
+/*
```

Comment:
What specific user would need to be able to look at a rule in our JSON Schemas?  My assumption was the opposite, that end users shouldn't even be aware that we're using JSON Schemas at all.